.github/workflows/main.yml

```diff
@@ -22,15 +22,15 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
-    - name: Test with pytest
```

Comment:
Some of these changes look unrelated, should they be reverted?

Reply:
Oops, I commented out those steps to speed up the testing process but that's not needed anymore.